asilib/asi/themis.py

```diff
@@ -59,12 +59,16 @@ def themis(
         If True, will download the data from the internet, regardless of
         wether or not the data exists locally (useful if the data becomes
         corrupted).
-    custom_alt: bool
-        If True, asilib will calculate (lat, lon) skymaps assuming a spherical Earth. Otherwise, it will use the official skymaps (Courtesy of University of Calgary).
+    custom_alt: str, default None
+        When selected, there are two options for skyma's between official sky maps:
+        If 'Geodetic', asilib will calculate (lat, lon) skymaps assuming a spherical Earth. Otherwise, it will use the official skymaps (Courtesy of University of Calgary).
```

Comment:
Can you change 'Geodetic' to 'geodetic'? If I pass 'Geodetic' into this function it will raise an error.